app/src/redux/analytics/constants.ts

```diff
@@ -45,3 +45,10 @@ export const ANALYTICS_ODD_APP_ERROR = 'oddError'
 export const ANALYTICS_NOTIFICATION_PORT_BLOCK_ERROR =
   'notificationPortBlockError'
 export const ANALYTICS_DESKTOP_APP_ERROR = 'desktopAppError'
+
+export const ANALYTICS_ROBOT_UPDATE_VIEW = 'robotUpdateView'
+export const ANALYTICS_ROBOT_UPDATE_CHANGE_LOG_VIEW = 'robotUpdateChangeLogView'
+export const ANALYTICS_ROBOT_UPDATE_IGNORE = 'robotUpdateIgnore'
+export const ANALYTICS_ROBOT_UPDATE_INITIATE = 'robotUpdateInitiate'
+export const ANALYTICS_ROBOT_UPDATE_ERROR = 'robotUpdateError'
+export const ANALYTICS_ROBOT_UPDATE_COMPLETE = 'robotUpdateComplete'
```

Comment:
Nit, I think we could benefit from doing an object literal here like the following:

```suggestion
const ANALYTICS_ROBOT_UPDATE = {
  VIEW: 'robotUpdateView',
  CHANGE_LOG_VIEW: 'robotUpdateChangeLogView',
  IGNORE: 'robotUpdateIgnore',
  INITIATE: 'robotUpdateInitiate',
  ERROR: 'robotUpdateError',
  COMPLETE: 'robotUpdateComplete',
} as const;
```
This gives us the added benefit of not importing the entire and enforces the relationship among these constants.

Reply:
thank you @mjhuff, that looks good.
I will update them.

Reply:
@mjhuff 

Do you think we should do the same change to run events?

```ts
export const ANALYTICS_PROTOCOL_RUN_AGAIN = 'runAgain'
export const ANALYTICS_PROTOCOL_RUN_FINISH = 'runFinish'
export const ANALYTICS_PROTOCOL_RUN_PAUSE = 'runPause'
export const ANALYTICS_PROTOCOL_RUN_START = 'runStart'
export const ANALYTICS_PROTOCOL_RUN_RESUME = 'runResume'
export const ANALYTICS_PROTOCOL_RUN_CANCEL = 'runCancel'

const ANALYTICS_PROTOCOL_RUN = {
  AGAIN: 'runAgain'
  FINISH: 'runFinish',
  PAUSE: 'runPause',
  START: 'runStart',
  RESUME: 'runResume',
  CANCEL: 'runCancel'
} as const


```

Reply:
Good catch - yes!